.github/workflows/repo-sync.yml

```diff
@@ -18,18 +18,10 @@ jobs:
       run: sudo chmod -R ugo+rwX .
     - name: Git add user.email & user.name
       run: git config --global user.email "nc-duenkekl3@netcologne.de" && git config --global user.name "Martin Dünkelmann"
-    - name: Rebase git remote update
-      run: git remote update
-    - name: Rebase git fetch
-      run: git fetch
```

Comment:
These commands do the same. But `repo sync` should have already fetched for you to checkout the repo in the first place.